interfaces/seccomp/backend.go

```diff
@@ -174,20 +199,27 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	if err := os.MkdirAll(dir, 0755); err != nil {
 		return fmt.Errorf("cannot create directory for seccomp profiles %q: %s", dir, err)
 	}
-	_, _, err = osutil.EnsureDirState(dir, glob, content)
+	changed, removed, err := osutil.EnsureDirState(dir, glob+".src", content)
```

Comment:
Could you move the `+".src"` part to the glob definition a few lines above please?